docker/btcd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine as builder
```

Comment:
btw - I'm not sure what's up with githubs diffing here, but this is what's on master